### ISSUE TITLE
Hiding Spines

### ISSUE THREAD

**Issue by joel-e-m-mitchell** [rougier/matplotlib-tutorial#9]
Hi, thank you for your tutorial, it's been really helpful!

In the [Moving Spines](https://github.com/rougier/matplotlib-tutorial#moving-spines) section, you've used `ax.spines['right'].set_color('none')` to hide the axis.
I wonder if `ax.spines['right'].set_visible(False)` would be a more elegant solution?

**Comment by rougier** [comment]
You're right actually. I wonder if this makes a difference for the ticks (do they become invisible as well with the invisible property)?
